src/fromager/commands/step.py

```diff
@@ -66,6 +64,27 @@ def prepare_source(wkctx, dist_name, dist_version):
     print(source_root_dir)
 
 
+@step.command()
+@click.argument("dist_name")
+@click.argument("dist_version")
+@click.pass_obj
+def build_sdist(wkctx, dist_name, dist_version):
+    """build a new source distribution for the package
+
+    DIST_NAME is the name of a distribution
+
+    DIST_VERSION is the version to process
+
+    The source distribution is placed in the `sdists-repo/builds`
+    directory.
+
+    """
+    req = Requirement(f"{dist_name}=={dist_version}")
+    source_root_dir = _find_source_root_dir(wkctx.work_dir, req, dist_version)
+    sdist_filename = sources.build_sdist(wkctx, req, source_root_dir)
+    print(sdist_filename)
```

Comment:
debug print?

```suggestion
    return sdist_filename
```

Reply:
The commands print the thing they create as output so that scripts can save the values to do something with them.